### PR TITLE
refs task/5842 Cart page

### PR DESCRIPTION
**Pull request by NamHuy18**
Trang giỏ hàng
https://edu-redmine.sun-asterisk.vn/issues/5842

### CODE REVIEW

**Comment by NamHuy18** [comment]
ready

File: resources/views/front/layouts/header.blade.php [hunk 1]
```diff
@@ -40,110 +40,73 @@
...
+                                            <span class="value">{{ Cart::total() }}</span>
                                         </div>
                                     </div>
-                                    <div class="min_cart_checkout checkoyt_two">
-                                        <a href="checkout.html">{{ trans('setting.checkout') }}</a>
+                                    <div class="min_cart_checkout">
+                                        <a href="{{ route('listCart') }}">{{ trans('setting.cart') }}</a>
                                     </div>
-                                </div>
+                                </div>
```
Comment (BunnyPi04) [inline resources/views/front/layouts/header.blade.php]: xóa space dư

**Comment by NamHuy18** [comment]
ready

File: routes/web.php [hunk 2]
```diff
@@ -89,4 +89,11 @@
 Route::get('food/{id}', 'PageController@food')->name('food');
 Route::post('comment/{id}', 'CommentController@postComment')->name('comment');
 Route::get('category/{id}', 'PageController@category')->name('category');
+Route::group(['prefix' => 'cart'], function()
+{
+	Route::post('add/{id}', 'CartController@addCart')->name('addCart');
```
Comment (BunnyPi04) [inline routes/web.php]: indent = 4 spaces nhé

**Comment by NamHuy18** [comment]
ready

**Pull request merged**